### ISSUE TITLE
npm scripts using 'export' throws errors in windows environment

### ISSUE THREAD

**Issue by briancodes** [sandrina-p/essential-webpack-boilerplate#5]
The following scripts cause errors in Windows 10 

```
"build": "export NODE_ENV=production && webpack --config ./config/webpack.config.js -p --optimize-minimize",
    "start": "export NODE_ENV=dev && webpack-dev-server --config ./config/webpack.config.js",
    "test": "export NODE_ENV=test && node --harmony_proxies node_modules/jest/bin/jest.js --coverage"  
```
The ```export``` command is not recognised. I have added a fix locally, using ```cross-env``` package - if you wish I can pr it to a new branch for you to test - it's a minimal change.

One other thing: is the ```&&``` required before ```webpack``` command in those scripts? Didn't think that would work. I removed them to get the environment vars working in my setup. 

Good boilerplate btw. Thanks

**Comment by sandrina-p** [comment]
A PR would be very welcome. When I find the time, I'll update the boilerplate and fix it. 
Thank you for reporting the issue 👍

**Comment by sandrina-p** [comment]
Fixed at #6
